charts/efc/templates/_helpers.tpl

```diff
@@ -43,13 +43,14 @@ Create chart name and version as used by the chart label.
           {{- $parts := splitList "," .mediumtype }}
           {{- range $i, $val := $parts }}
             {{- /* Example: For path="/tmp/mem,/tmp/ssd", mountPath resolves to /tmp/mem and /tmp/ssd */}}
+            {{- $mediumName := printf "%v-%v" (lower $val) $i }}
             - mountPath: {{ index ($path | splitList ",") $i }}
-              name: {{ $val | lower }}-{{ $i }}
+              name: efc-{{ $mediumName }}
```

Comment:
I suggest not changing helpers.tpl  which is for tieredStore in Worker. Instead, we just need change fuse.yaml to make it does nothing with tieredstore.